widgets/common/src/deploymentsView/DeploymentsView.tsx

```diff
@@ -21,28 +21,53 @@ export interface DeploymentsViewProps {
     toolbox: Stage.Types.Toolbox;
 
     filterByParentDeployment: boolean;
-    filterRules: Stage.Common.Filters.Rule[];
-    fetchingRules: boolean;
+    defaultFilterId?: string | undefined;
```

Comment:
Maybe instead of `default` we could use `initial`. IMO that better describes the intention of the prop

```suggestion
    initialFilterId?: string | undefined;
```

btw. do we need both `?` and `| undefined`? I think just `?` works well, as far as I have checked locally
```suggestion
    initialFilterId?: string;
```

Reply:
> Maybe instead of `default` we could use `initial`. IMO that better describes the intention of the prop

As for me:
- initial - just a starting point
- default - value that is used when there is no explicitly specified value, it's common to revert to default

So I think 'default' suites better.

> btw. do we need both `?` and `| undefined`? I think just `?` works well, as far as I have checked locally

Yes, updated.

Reply:
You're right about the default vs initial dilemma :+1: Thanks for the explanation